### PR TITLE
update links

### PR DESCRIPTION
**Pull request by rianfowler**
## Description
add additional links to account page on `va.gov`
[issue](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/12617)

## Testing done
verified locally 

## Screenshots va.gov
screen shot 2018-10-01 at 15 31 07
screen shot 2018-10-01 at 15 31 03
screen shot 2018-10-01 at 14 52 57

Note: separate  issues have been filed to address
- weird e styling in MyHealtheVet [link](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/13857)
- missing icon on accordion [link](https://app.zenhub.com/workspace/o/department-of-veterans-affairs/vets.gov-team/issues/13856)

## screenshots vets.gov
screen shot 2018-10-01 at 15 29 53

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/account/components/AccountMain.jsx [hunk 1]
```diff
@@ -72,6 +72,17 @@ class AccountMain extends React.Component {
         <AccountVerification loa={loa}/>
         {this.renderMVIError()}
         <MultifactorMessage multifactor={multifactor}/>
+        {isBrandConsolidationEnabled &&
+          <div>
```
Comment (erikphansen) [inline src/applications/personalization/account/components/AccountMain.jsx]: I don't _think_ you need this wrapper `div`

File: src/applications/personalization/account/components/LoginSettings.jsx [hunk 2]
```diff
@@ -1,20 +1,30 @@
...
+          <p><a href="https://wallet.id.me/settings" target="_blank">Manage your ID.me account.</a></p>
+          <AdditionalInfo
+            triggerText="What is ID.me?">
+            {idMeAnswer}
+          </AdditionalInfo>
+        </div>) :
+        (<p>Want to change the email you use to sign in to ID.me, or update your ID.me password or other account settings?<br/>
+          <a href="https://wallet.id.me/settings" target="_blank">Go to ID.me to manage your account</a>
+          {idMeAnswer}
+        </p>)}
```
Comment (erikphansen) [inline src/applications/personalization/account/components/LoginSettings.jsx]: I think this closing `p` tag should come right after the `a` tag and before the inclusion of the `idMeAnswer` content.

**Review by erikphansen — APPROVED** [review]
Just a couple suggestions.

File: src/applications/personalization/account/components/AccountMain.jsx (hunk 1 above)
Comment (rianfowler) [reply src/applications/personalization/account/components/AccountMain.jsx]: The `{ }` has to return a single React node or an array type or it won't compile.

Comment (rianfowler) [reply src/applications/personalization/account/components/AccountMain.jsx]: or null*

**Comment by Samara-Strauss** [comment]
Hello! Just QA'd this on staging.va.gov. All links work, and things mostly look good. However, I did notice the following:

* The heading "Sign in settings" is missing, as is its corresponding description.
  * **Heading copy**: Sign in settings
  * **Description copy**: You can update the email or password you use to sign in to Vets.gov. Just go to the account you use to sign in (DS Logon, MyHealtheVet, or ID.me) and manage your settings.
*  The headings "DS Logon," "MyHealtheVet," and "ID.me" should all be H5s under the "Sign in settings" heading, which is an H3.

Below is a screenshot which should help clarify the above:

account-full-v2

**Pull request merged**